features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/api/api.jag

```diff
@@ -55,6 +55,51 @@ var getTiers = function () {
     }
 };
 
+var getResourceTiers = function () {
+    var tiers,
+            log = new Log();
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea